labellab-flask/api/controllers/imagescontroller.py

```diff
@@ -0,0 +1,215 @@
+from datetime import datetime
+
+from flask.views import MethodView
+from flask import make_response, request, jsonify, current_app
+from flask_jwt_extended import (
+    jwt_required,
+    get_jwt_identity,
+    get_raw_jwt
+)
+from api.models.Image import Image
+from api.helpers.image import (
+    convert_and_save,
+    get_dimensions,
+    save as save_image,
+    find_all_by_project_id as find_images,
+    find_by_id,
+    remove_image,
+    delete_by_id
+)
+
+
+class SubmitImage(MethodView):
+    """This class saves a new image."""
+    @jwt_required
+    def post(self, project_id):
+        """
+        Handle POST request for this view.
+        Url --> /api/v1/image/create/<int:project_id>
+        """
+        # getting JSON data from request
+        post_data = request.get_json(silent=True,
+                                     force=True)
+        current_user = get_jwt_identity()
+        try:
+            images = post_data["images"]
+            image_names = post_data["image_names"]
+            format = post_data["format"]
+
+        except KeyError as err:
+            response = {
+                "success": False,
+                "msg": f'{str(err)} key is not present'
+            }
+            return make_response(jsonify(response)), 400
+
+        project_id = project_id
+        user_id = current_user
```

Comment:
its current_user.id right?

Reply:
no no current_user is the user_id only 🙂

Reply:
oh ok